app/tests/components/MainSection-test.js

```diff
@@ -1,43 +1,33 @@
 import React from 'react';
-import ReactTestUtils from 'react-addons-test-utils';
 import expect from 'expect';
-import { wrap } from 'react-stateless-wrapper';
+import { mount } from 'enzyme';
 import MainSection from 'components/MainSection';
-import TopicItem from 'components/TopicItem';
 
-const WrappedMainSection = wrap(MainSection);
+const emptyData = [];
+const topicItemData = [{
+  text: '',
+  id: '',
+  index: 0,
+  onIncrement: () => {},
+  onDecrement: () => {},
+  onDestroy: () => {}
+}];
+const stubFunctions = {
+  onIncrement: () => {},
+  onDecrement: () => {},
+  onDestroy: () => {}
+};
 
-describe('MainSection', () => {
-  let result;
-  let topicItems;
-  const topicItemData = {
-    text: '',
-    id: '',
-    index: 0,
-    onIncrement: () => {},
-    onDecrement: () => {},
-    onDestroy: () => {}
-  };
-  const stubFunctions = {
-    onIncrement: () => {},
-    onDecrement: () => {},
-    onDestroy: () => {}
-  };
-  const topics = [topicItemData];
-
-  describe('Has topics', () => {
-    it('should render TopicItems', () => {
-      result = ReactTestUtils.renderIntoDocument(<WrappedMainSection topics={topics} {...stubFunctions} />);
-      topicItems = ReactTestUtils.scryRenderedComponentsWithType(result, TopicItem);
-      expect(topicItems.length).toBe(1);
+describe('<MainSection />', () => {
+  describe('With Topics', () => {
+    it('should render <TopicItem> list items', () => {
+      expect(mount(<MainSection topics={topicItemData} {...stubFunctions} />).find('.list li').length).toBe(1);
```

Comment:
Minor comment: should this be `find(TopicItem)` given that we are asserting that `MainSection` contains `TopicItem`? Finding `.list li` works, but that ties this test to the specific implementation of `TopicItem`, i.e. if at some point in the future, we change TopicItem to be `<div>`, this test will break even though it shouldn't. :)